language/Turkish/strings.po

```diff
@@ -6341,72 +6364,72 @@ msgstr ""
 #: plugins/timer/init.lua:32
 msgctxt "plugin-timer"
 msgid "Wall clock"
-msgstr ""
+msgstr "Duvar saati"
```

Comment:
In this context, “Wall clock” refers to elapsed real time using an independent clock/time reference.  Can you confirm that the Turkish translation is appropriate?

Reply:
Will compile Mame tomorrow and check all the translations in place.
A noob question: do i have to make new pull request for the corrections?

Reply:
UI translation files are a bit outdated at the moment, they aren’t quite in sync with the application.  I need to get it back in sync, but I also need to review the Brazilian Portuguese updates before I can do that.  The “Source File” filter was added since the last time I got the UI translation files in sync, so it can’t be localised at the moment.  Sorry.

Reply:
that's ok.
meanwhile will check in place transilations from time to time to fix any misuses.Thanks.